egs/public_dataset/jsut-song_ver1/README.md

```diff
@@ -0,0 +1,16 @@
+# JSUT-song
+The JSUT Collection is Japanese speech corpora connecting speech, song, and audio events. The JSUT-song corpus is a part of the JSUT Collection.
+
+## Running Instruction
+
+- Train: `sh train.sh` 
+- Infer: `sh infer.sh` 
+
+Related configuration files for different models are under the folder 'conf'
+
+## Resouces
+- Wav files: <https://www.google.com/url?q=https%3A%2F%2Fss-takashi.jp%2Fcorpus%2Fjsut-song_ver1.zip&sa=D&sntz=1&usg=AFQjCNFgxk6QKvG1nsqoazpMQRT2_l65BA>
```

Comment:
If the corpus would be OK with `wget` or other stuff, please directly download it in `run.sh`

If from google drive, [this](https://github.com/espnet/espnet/blob/master/utils/download_from_google_drive.sh) would be helpful